dev/io.openliberty.jakarta.validation.v31_fat/test-applications/val31/src/val31/web/Person.java

```diff
@@ -17,10 +17,8 @@
  *
  */
 public record Person(@NotNull String name) {
+    public String checkNameSize(@Size(max = 10) String x) {
 
-    @Size(min = 6)
-    public String getName() {
-        return this.name;
+        return x + "String value";
     }
```

Comment:
We still want to validate the return value, so the getName method is still needed

Reply:
Updated